WcaOnRails/app/views/server_status/index.html.erb

```diff
@@ -62,6 +62,8 @@
     <span>PID:</span> <%= Process.pid %>
     <%= icon "circle" %>
     <span>Uptime:</span> <%= duration_to_s(Time.now - WcaOnRails::BOOTED_AT) %>
+    <%= icon "circle" %>
+    <a href="https://github.com/thewca/worldcubeassociation.org/commit/#{`git rev-parse HEAD`}">Git commit</a>
   <% end %>
```

Comment:
That is a clear improvement! Especially that you can see the short sha on the page. The lack of ERB syntax debris is also real nice.